### PR TITLE
[Changed][BEL-12075] Update readme with pagination

### PR DESCRIPTION
**Pull request by matheuslins**
:question: What
---
- Readme updated

:speech_balloon: Why
---
We need to explain how to paginate after got the results from the `list` method

### CODE REVIEW

**Comment by matheuslins** [comment]
Note: I didn't want to set a specific way to iterate over pages because it's something that belongs to each client

File: README.md [hunk 1]
```diff
@@ -94,6 +94,17 @@ rescue Belvo::RequestError => e
 end
 ```
 
+## Pagination ##
```
Comment (dom-techwriter) [inline README.md:97]:
```suggestion
## Pagination
```

File: README.md [hunk 2]
```diff
@@ -94,6 +94,17 @@ rescue Belvo::RequestError => e
...
+
+All `list` methods only return the first page of results. If you want to iterate by all pages, you will need to specify the `page` as a parameter
```
Comment (dom-techwriter) [inline README.md:99]:
```suggestion
All `list` methods only return the first page of results. If you want to iterate by all pages, you will need to specify the `page` as a parameter:
```

File: README.md [hunk 3]
```diff
@@ -94,6 +94,17 @@ rescue Belvo::RequestError => e
...
+
+```ruby
+    #`api/transactions`
+    ...
+    belvo.transactions.list(page: "2")
+    ...
+```
+
```
Comment (dom-techwriter) [inline README.md:107]:
```suggestion
When the result of your call is `nil`, this indicates that there are not more pages to iterate over. 
```

File: README.md [hunk 4]
```diff
@@ -94,6 +94,18 @@ rescue Belvo::RequestError => e
...
+All `list` methods only return the first page of results. If you want to iterate by all pages, you will need to specify the `page` as a parameter:
+
+```ruby
+    #`api/transactions`
+    ...
+    belvo.transactions.list(params: {page: "2"})
+    ...
+```
+When the result of your call is `nil`, this indicates that there are not more pages to iterate over.
```
Comment (dom-techwriter) [inline README.md:107]:
```suggestion
When the result of your call is `nil`, this indicates that there are no more pages to iterate over. 
```

**Review by dom-techwriter — COMMENTED** [review]
I made a stupid typo in my suggestion...

**Review by AlbertMaristany — APPROVED** [review]

**Review by dom-techwriter — APPROVED** [review]
lgtm!

**Pull request merged**